hathor/crypto/util.py

```diff
@@ -132,6 +129,8 @@ def get_address_from_public_key_hash(public_key_hash: bytes,
         :return: address in bytes
         :rtype: bytes
     """
+    settings = get_settings()
+    version_byte = version_byte if version_byte is not None else settings.P2PKH_VERSION_BYTE
```

Comment:
Maybe explicitly add the type `version_byte: bytes`.

Reply:
The linter wouldn't let me do it like this, so I just changed the format of the `if` statement in https://github.com/HathorNetwork/hathor-core/pull/769/commits/599cc4f39d6fce979ab57098126914347c89fff8

Reply:
Changed in https://github.com/HathorNetwork/hathor-core/pull/769/commits/a6a965b0448ce7ab8ac5ae7f1b119b76898d469f. I had to change the variable name for the linter to accept it.

Reply:
Thanks!